remix-analyzer/src/solidity-analyzer/modules/etherTransferInLoop.js

```diff
@@ -0,0 +1,31 @@
+var name = 'Ether transfer in a loop: '
+var desc = 'Avoid transferring Ether to multiple addresses in a loop'
+var categories = require('./categories')
+var common = require('./staticAnalysisCommon')
+
+function etherTransferInLoop () {
+  this.relevantNodes = []
+}
+
+etherTransferInLoop.prototype.visit = function (node) {
+  if (common.isLoop(node) && common.isTransfer(node)) {
```

Comment:
does that look for a `Transfer` inside a loop?

Reply:
Oh, something looks incorrect. I need to find Transfer inside loop.